Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-urls.tsx

```diff
@@ -1,12 +1,29 @@
 import React, { createContext, useContext } from 'react';
 import { isNil } from 'lodash';
-import { IAllContestsUrlParams, IStartContestUrlParams } from '../common/url-types';
+import {
+    IAllContestsUrlParams,
+    IDownloadProblemResourceUrlParams, IGetCurrentSubmissionDetailsUrlParams, IGetSubmissionResultsByProblemUrlParams,
+    IStartContestParticipationUrlParams,
+} from '../common/url-types';
 import { IHaveChildrenProps } from '../components/common/Props';
 import { FilterType } from '../common/contest-types';
 
 interface IUrlsContext {
-    getUrlForStartContestParticipation: (params: IStartContestUrlParams) => string;
-    getUrlForAllContests: (params: IAllContestsUrlParams) => string;
+    getLoginSubmitUrl: string;
+    getLogoutUrl: string;
+    getAdministrationContestsGridUrl: string;
+    getProfileInfoUrl: string;
+    getSubmissionsForProfileUrl: string;
+    getParticipationsForProfileUrl: string;
+    getIndexContestsUrl: string;
```

Comment:
It's strange to have a variable called `get...` that is a string.

`get` implies action, i.e. a function.

Either make all these functions, or rename them. (Better to be functions)